### DISCUSSION TITLE
Put Parent Pages & Child Pages based on folder structure

### DISCUSSION THREAD

**Discussion by KyleMHB** [markdown-confluence/markdown-confluence#170] [Ideas]
@andymac4182 would you have any apatite for your plugin to be able to publish into a structure based on the folder structure?

For Example
 \Confluence Pages
   \Customers
     \Customer One
       \Project One
         Proposal.md
         Notes.md

The the parent pages don't even need to have any content, they could just be placeholders for people to add maybe a filetree to or something

It would allow me to have a structured wiki for any work Im doing without having to manually move stuff and create parent pages. It would be really great

**Comment by KyleMHB** [comment]
NVM I was just being an idiot, this already works.

**Comment by andymac4182** [comment]
If you combine the folder structure with https://obsidian-confluence.com/features/folder-note.html you can create the files that will be the folder in the tree :) 

https://obsidian-confluence.com/features/folder-structure.html

There is one feature required if you are moving files around in confluence as well where the tree is based on your local file system not what has moved in confluence already. Will have a look at fixing that in the future.
